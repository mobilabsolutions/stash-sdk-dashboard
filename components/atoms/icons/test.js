/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import { withTestSetup } from '../../../test_utils'
import DownIcon from './down'
import UpIcon from './up'

const TestDownIcon = withTestSetup(DownIcon)
const TestUpIcon = withTestSetup(UpIcon)

it('Down Icon Should Render', () => {
  const component = renderer.create(<TestDownIcon />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

it('UpIcon Should Render', () => {
  const component = renderer.create(<TestUpIcon />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
