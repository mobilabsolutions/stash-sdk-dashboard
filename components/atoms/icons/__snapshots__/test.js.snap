// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Down Icon Should Render 1`] = `
<svg
  className="enable"
  height={20}
  viewBox="0 0 48 48"
  width={20}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M14.83 16.42L24 25.59l9.17-9.17L36 19.25l-12 12-12-12z"
  />
</svg>
`;

exports[`UpIcon Should Render 1`] = `
<svg
  className="enable"
  height={20}
  viewBox="0 0 48 48"
  width={20}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M14.83 30.83L24 21.66l9.17 9.17L36 28 24 16 12 28z"
  />
</svg>
`;
